src/modules/mod.rs

```diff
@@ -71,10 +84,12 @@ impl App {
         modules_def
             .iter()
             .flat_map(|module_def| match module_def {
-                ModuleDef::Single(module) => vec![self.get_module_subscription(*module)],
+                ModuleDef::Single(module) => {
+                    vec![self.get_module_subscription(module.to_owned().into())]
```

Comment:
Well... there are some other places where we should fix this kind of stuff. But in this way we can avoid some `clone`

```suggestion
                    vec![self.get_module_subscription(module)]
```